shared/src/main/kotlin/com/egm/stellio/shared/model/ExpandedEntity.kt

```diff
@@ -53,12 +53,13 @@ data class ExpandedEntity(
                 if (JSONLD_EXPANDED_ENTITY_CORE_MEMBERS.contains(it.key))
                     it.value
                 else castAttributeValue(it.value).map { expandedAttributeInstance ->
-                    expandedAttributeInstance.addDateTimeProperty(
-                        NGSILD_CREATED_AT_PROPERTY,
-                        createdAt
-                    ) as ExpandedAttributeInstance
+                    expandedAttributeInstance
```

Comment:
as the name of the function says, it is called at entity creation time, but you are missing attributes creation times (and entity replace?).